src/css/imports/controlbar.less

```diff
@@ -14,13 +14,22 @@
     }
 }
 
+.jw-breakpoint-0 .jw-controlbar {
```

Comment:
Another problem with `.inset-controlbar()` is that it sets `display: inline-block;` when the controlbar should not be displayed while the player is idle.

```css
.inset-controlbar() {
    .jw-controlbar {
        // for correct display in IE browsers
        display: inline-block;
        // width assignment is required for IE11 to center correctly
        width: 96%;
        max-width: 50em;
        margin: 0 auto;
        bottom: .7em;
        left: 2%;
        right: 2%;
    }
    &.jw-flag-audio-player .jw-controlbar {
        bottom: 0; // overwrite the inset-controlbar mixin
    }
}
```